### ISSUE TITLE
Tampermonkey 4.6 Broke Accessibility for Chrome

### ISSUE THREAD

**Issue by chigkim** [Tampermonkey/tampermonkey#564]
I was able to manage custom scripts using a screen reader in 4.6.
However, my screen reader cannot select to edit/delete scripts in 4.6. It seems like the buttons are not accessible screen reader anymore.
How can I go back to 4.5 until it gets fixed?
Thanks,

**Comment by derjanb** [comment]
Duplicate of #1185
